src/utils/interfaces.ts

```diff
@@ -0,0 +1,22 @@
+export interface FormConfig {
```

Comment:
Daca interfata tine de formGenerator, poate fi un file in componenta, formGenerator. Asta daca nu cumva o sa mai folosim interfata in alta componenta.

Reply:
Done